docs/designers-developers/developers/tutorials/format-api/2-toolbar-button.md

```diff
@@ -35,3 +35,49 @@ Let's check that everything is working as expected. Reload the post/page and sel
 ![Toolbar with custom button](https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/designers-developers/assets/toolbar-with-custom-button.png)
 
 You may also want to check that upon clicking the button the `toggle format` message is shown in your browser's console.
+
+By default, the button is rendered on every rich text toolbar (image captions, buttons, paragraphs, etc).
+It is possible to render the button only on blocks of a certain type by using `wp.data.withSelect` together with `wp.compose.ifCondition`.
```

Comment:
I'd like us to add a reminder that to use `wp.data` and `wp.compose` the authors needs to add them to the plugin dependencies.